pom.xml

```diff
@@ -45,6 +45,11 @@
 			<artifactId>liquibase-core</artifactId>
 			<version>4.2.2</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
```

Comment:
spring-boot-starter-test already add junit5 to the dependencies, can you pleease remove this one?